assets/promptflow/evaluators/models/gleu-score-evaluator/spec.yaml

```diff
@@ -0,0 +1,11 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: Gleu-Score-Evaluator
+path: ./
+properties:
+  is-promptflow: true
+  is-evaluator: true
+  show-artifact: true
+  _default-display-file: ./evaluator/_gleu.py
```

Comment:
this will show the _gleu.py file when user lands on the evaluator. Is this by design?

Reply:
same question for other text similarity evaluators.

Reply:
Yes, I've sent the test registry for Wenxin and Minsoo to review ealier. It looks good for them.

https://ai.azure.com/build/evaluation/evaluator?wsid=/subscriptions/e0fd569c-e34a-4249-8c24-e8d723c7f054/resourceGroups/rg-qunsongai/providers/Microsoft.MachineLearningServices/workspaces/qunsong-0951&flight=ModelCatalogAMLTestRegistryName=RAI-Simulator&tid=72f988bf-86f1-41af-91ab-2d7cd011db47